frontend/AnimeCore/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,205 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+
+	const icon_mapping = {
+		anime_options: {
+			read: {
+				icon: {
+					component: Read,
+					width: 22,
+					height: 22,
+					color: "bg-surface-500"
+				}
+			},
+			listen: {
+				icon: {
+					component: Listen,
+					width: 22,
+					height: 22,
+					color: "bg-surface-500"
+				}
+			},
+		},
+		user_options_icons : {
+			video: {
+				icon: {
+					component: Video,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			},
+			edit: {
+				icon: {
+					component: Edit,
+					variant: "with_underline_around_pencil",
+					width: 20,
+					height: 20
+				}
+			},
+			download: {
+				icon: {
+					component: Download,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			},
+			share: {
+				icon: {
+					component: Share,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			}
+		}
+	}
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end md:gap-10">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="md:w-[15vw] md:rounded-xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<span class="font-bold md:text-[3vw]">{anime.titles.eng}</span>
+
+				<p class="flex items-center gap-2 text-surface-100">
+					{#each Object.entries(anime.titles) as anime_title}
+						{#if anime_title[0] !== 'eng'}
+							<span class="md:text-[0.8vw] font-medium [&:not(:last-child)]:after:content-['▪'] [&:not(:last-child)]:after:ml-2">{anime_title[1]}</span>
+						{/if}
+					{/each}
+				</p>
+
+				<p class="pt-2 flex flex-wrap items-center gap-2">
+					{#each Object.entries(anime.basic_details) as anime_detail}
+						<span class="md:text-[0.9vw] md:leading-none font-semibold [&:not(:last-child)]:after:content-['▪'] [&:not(:last-child)]:after:ml-1">
+							{anime_detail[1]} {anime_detail[0] === 'episodes' ? 'eps' : ''}
+						</span>
+					{/each}
```

Comment:
>							`{anime_detail[1]} {anime_detail[0] === 'episodes' ? 'eps' : ''}`

What does this do?

Reply:
It add 'eps' for only value with key 'episodes'

Reply:
> It add 'eps' for only value with key 'episodes'


This approach is not scalable tbh. I will refactor it